test/fail_compilation/test19097.d

```diff
@@ -1,7 +1,8 @@
 /* REQUIRED_ARGS: -preview=dip1000
  * TEST_OUTPUT:
 ---
-fail_compilation/test19097.d(35): Error: scope variable `s` may not be returned
+fail_compilation/test19097.d(36): Error: scope variable `s` may not be returned
+fail_compilation/test19097.d(74): Error: scope variable `s` may not be returned
```

Comment:
Shouldn't this error refer to `s.i` instead of `s`?

Reply:
escape.d likes to reduce expressions to 'the variable it really is', so s.x => s`, or `sarray[0] => sarray` etc. This is reflected in the error messages.